usecases/service/group_test.go

```diff
@@ -0,0 +1,109 @@
+package service
+
+import (
+	"context"
+	"testing"
+
+	"github.com/traPtitech/traPortfolio/util/random"
+
+	"github.com/gofrs/uuid"
+	"github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/assert"
+	"github.com/traPtitech/traPortfolio/domain"
+	"github.com/traPtitech/traPortfolio/usecases/repository/mock_repository"
+)
+
+func TestGroupService_GetAllGroups(t *testing.T) {
+	t.Parallel()
+	type args struct {
+		ctx context.Context
+	}
+	tests := []struct {
+		name      string
+		args      args
+		want      []*domain.Group
+		setup     func(repo *mock_repository.MockGroupRepository, user *mock_repository.MockUserRepository, args args, want []*domain.Group)
```

Comment:
```suggestion
		setup     func(group *mock_repository.MockGroupRepository, user *mock_repository.MockUserRepository, args args, want []*domain.Group)
```
多分自動生成してもらったと思うんですが`repo`は将来消す予定なので`group`にしてもらえると:pray:
他のとこもお願いします